Dockerfile.multi-stage

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.13.6-alpine3.11 as builder
+FROM golang:1.14.2-alpine3.11 as builder
```

Comment:
not sure how to use `node` one here (:

Reply:
What version of node do we need here? and yarn?

Reply:
cc @prmsrswt

Reply:
Not a blocker I guess.

Reply:
I don't think we need `node` here. It doesn't use node and yarn to build the React UI, rather it uses the `bindata.go` file to get pre-compiled react assets.